se.data/langs/es_la.json

```
@@ -0,0 +1,171 @@
+  "more_moves_undo": "w Valor demasiado grande, deshaciando todas las acciones de %s",
+  "help_welcome_tooltip": "y Síii!",
+  "action_item_angel": "item de bloque ángel",
+  "saved_structure": "w Estructura guardada como %s.nbt",
+  "help_cmd_rotate": "l Rota [deg] en torno a [pos]",
+  "brush_info_action": "b  Acción: ",
+  "help_cmd_brush_pyramid": "l Registra brocha para crear pirámide con [radius] y [height] sobre eje [axis] en la dirección del signo",
+  "langs_completeness": "gi Nota: %s la traducción está completa en un %s%%: %s frases faltantes",
+  "no_undo": "r Sin acciones que deshacer de %s",
+  "help_cmd_brush_outline": "l Registra brocha de contorno",
+  "help_cmd_rotate_tooltip": "g <axis> (eje) debe ser x, y o z",
+  "angel_block_none_error": "r Aún no hay bloque ángel definido! Usar 'angel new' para registrar uno",
+  "help_cmd_move": "l Mueve la selección a pos <pos>",
+  "langs_changed": "gi Idioma cambió a %s.",
+  "copy_force": "ri Sobreescribiendo portapapeles anterior con el nuevo",
+  "action_ellipsoid": "elipsoide",
+  "help_cmd_brush_cuboid": "l Registra brocha para crear un cuboide de dimensiones [x] [y] y [z] de [block]",
+  "entry_undo_1": "w %d: tipo: %s (dimensión: %s):",
+  "brush_reach": "w El alcance de las brochas es %d bloques",
+  "help_cmd_undo_history": "l Muestra la historia de acciones deshechas",
+  "success_redo": "gi Se rechicieron %d exitosamente, rellenando %d bloques",
+  "no_longer_brush": "w %s ya no es más una brocha",
+  "help_cmd_undo": "l Deshace los últimos <n> movimientos, uno sólo por defecto",
+  "brush_info_flags": "b Banderas: ",
+  "action_sphere": "esfera",
+  "help_cmd_paste": "l Pega portapapeles",
+  "help_cmd_brush_generic": "l Sostiene un ítem para covnertirlo en brocha",
+  "angel_block_bad_item": "r Bloque nagel no puede ser alg que ya está asignado a otra acción",
+  "move_selection_no_player_error": "r Para mover la selección en la dirección del jugadro, hace falta un jugador",
+  "language": "Español",
+  "copy_success": "gi Se copiaron exitosamente %s bloques y %s entidadaes al portapapeles",
+  "structure_delete_success": "gi Se borró %s.nbt exitosamente",
+  "help_cmd_lang": "l Cambia el idioma de la aplicación a [lang]",
+  "help_list_title": "y Lista de comandos sin prefijos:",
+  "brush_item_tooltip": "^g Cliquea para obtener uno!",
+  "paste_no_clipboard": "ri No se pudo completar la acción, el portapapeles de %s está vacío",
+  "help_cmd_redo_tooltip": "g También aparece en la historia de acciones a deshacer",
+  "no_selection_error": "r Jugador %s no tiene selección disponible para la operación",
+  "action_cube": "cubo",
+  "new_wand": "wi La varita de la aplicación ahora es %s, usar con cuidado.",
+  "help_cmd_brush_spray_held": "l Registra brocha para pintura en aerosol con el bloque sostenido en la mano secundaria",
+  "help_welcome": "c Bienvenidos al menú de ayuda de Scarpet-Edit!",
```

I'd use singular. Not sure how to handle gender, though.
```suggestion
  "help_welcome": "c Bienvenido al menú de ayuda de Scarpet-Edit!",
```